tensorflow/c/c_api_function_test.cc

```diff
@@ -1462,7 +1462,11 @@ TEST_F(CApiFunctionTest, AppendHash) {
                  /*append_hash=*/true);
   tensorflow::FunctionDef fdef;
   ASSERT_TRUE(GetFunctionDef(func_, &fdef));
+#if ((__BYTE_ORDER__ == __ORDER_BIG_ENDIAN__) && defined(__s390x__))
```

Comment:
It seems like we can make the check more generic and remove " && defined(__s390x__)".